loopchain/launcher.py

```diff
@@ -116,7 +118,14 @@ def start_as_channel(args):
     amqp_target = args.amqp_target or conf.AMQP_TARGET
     amqp_key = args.amqp_key or conf.AMQP_KEY
 
-    ChannelService(channel, amqp_target, amqp_key).serve()
+    try:
+        ChannelService(channel, amqp_target, amqp_key).serve()
+    except ConsensusChanged as e:
+        utils.logger.info(f"Consensus Changed")
+        utils.logger.info(f"Remain txs. {len(e.remain_txs)}")
+        utils.logger.info(f"Last unconfirmed block {e.last_unconfirmed_block.header}")
+        utils.logger.info(f"Last unconfirmed votes {e.last_unconfirmed_votes}")
+        Node(e.node_id).start()
```

Comment:
I think new node needs all informations of ConsensusChanged exception. Will this pass in other feature?

Reply:
Yeah, Purpose of this PR is just for working without any problem with minimum number of modifications from original LFT `app`.